src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4488,7 +4488,6 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
 
                         - ADVANCED -
  /baseaddress:&lt;address&gt;        Base address for the library to be built
- /bugreport:&lt;file&gt;             Create a 'Bug Report' file
```

Comment:
Consider removing from VBResources.resx too. The option is no longer supported in VB either.